File: backend/src/main/java/com/team05/petmeeting/domain/comment/dto/AnimalCommentListRes.java
package com.team05.petmeeting.domain.comment.dto;

import java.util.List;

public record AnimalCommentListRes (
        List<AnimalCommentRes> comments,
        int totalCount
){
    public static AnimalCommentListRes from(List<AnimalCommentRes> comments){
        return new AnimalCommentListRes(comments, comments.size());
    }
}

File: backend/src/main/java/com/team05/petmeeting/domain/comment/dto/AnimalCommentListRes.kt
package com.team05.petmeeting.domain.comment.dto

data class AnimalCommentListRes(
    val comments: List<AnimalCommentRes>,
    val totalCount: Int
) {
    companion object {
        @JvmStatic
        fun from(comments: List<AnimalCommentRes>): AnimalCommentListRes {
            return AnimalCommentListRes(comments, comments.size)
        }
    }
}
